crates/indexer/tests/integration.rs

```diff
@@ -61,56 +61,6 @@ async fn test_indexer() -> Result<()> {
         .watch()
         .await?;
 
-    // InputPublished
```

Comment:
Might be good to replace this with a different event. This just happened to use this one as an example to test the indexer.

Reply:
Makes sense!